tests/_utils/utils.js

```diff
@@ -93,7 +48,8 @@ export function formatTable( tableString ) {
 		.replace( /<\/thead>/g, '\n</thead>' )
 		.replace( /<\/tbody>/g, '\n</tbody>' )
 		.replace( /<\/tr>/g, '\n</tr>' )
-		.replace( /<\/table>/g, '\n</table>' );
+		.replace( /<\/table>/g, '\n</table>' )
+		.replace( /<\/figure>/g, '\n</table>' );
```

Comment:
Is this a typo? (replacing "figure" with "table")

Reply:
Nice catch :D Thanks.

@jodator, I changed it and all tests passed. Are you sure this is used and needed?

Reply:
@Reinmar this is a helper method to nicely print out a table contents - it is used on both expected and actual results thus this slipped unnoticed...

This method helps a lot when some test fails and you read sinon output to easily spot what's wrong:

with `formatTable()`:

![selection_043](https://user-images.githubusercontent.com/247363/41656129-3c6c8c1e-748f-11e8-8389-b07f6a014b8b.png)

without:

![selection_044](https://user-images.githubusercontent.com/247363/41656135-411ddcd6-748f-11e8-9a22-74d0105e4a70.png)

I've also gradually updated other tests to use those helper methods as writing expected model table string is tedious....